### ISSUE TITLE
Raise in Android Crashes on version 5.1.2

### ISSUE THREAD

**Issue by dantasulisses** [AppLovin/AppLovin-MAX-Unity-Plugin#138]
Hello, after updating and releasing our game with the Unity App Lovin MAX version 5.1.2 we are seeing a raise in the following Crash, reported in GooglePlay Vitals and in Firebase Crashlytics:
```
java.lang.Error: FATAL EXCEPTION [AsyncTask #6]
Unity version     : 2020.3.25f1
Build Type        : Release
Scripting Backend : IL2CPP
ABI               : arm64-v8a
Strip Engine Code : true
Caused by
  at java.util.ArrayList.add (ArrayList.java:468)
  at org.json.JSONArray.put (JSONArray.java:187)
  at com.applovin.impl.mediation.c.c.a (Unknown Source:102)
  at com.applovin.impl.sdk.e.i.b (Unknown Source:171)
  at com.applovin.impl.sdk.e.i.run (Unknown Source:46)
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1167)
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:641)
  at java.lang.Thread.run (Thread.java:923)
```

We are not able to reproduce this locally on our devices, but it is happening quite often in the production release.
Is there any known fix?

**Comment by thomasmso** [comment]
@dantasulisses - Thanks for raising the issue, could you provide the exception message as well?

**Comment by dantasulisses** [comment]
Sorry @thomasmso , but I wasn't able to retrieve any more information about the error. Only this one, which is the error reported at Google Play Android Vitals

**Comment by thomasmso** [comment]
@dantasulisses - Could you send us _any_ more info? What is the app, and what version were you on previously that did _not_ have this crash?

**Comment by majunfei** [comment]
Encountered the same problem and looked forward to solving it

**Comment by thomasmso** [comment]
@majunfei - Could you send us any more info? Like the full exception message?

**Comment by dantasulisses** [comment]
Sure @thomasmso , some extra information:

- We were earlier at version 5.1.1, and this error only appeared in version 5.1.2
- The error happens in different Android versions and devices equally, so don't seem related to some specific Android version or device.
- Our game have 9 different Mediated-Ad-Networks turned on

Besides that, I still wasn't able to get any extra information or exception message

**Comment by dantasulisses** [comment]
Sure @thomasmso , I have these two very similar error names:
Error: java.lang.ArrayIndexOutOfBoundsException
```
java.lang.ArrayIndexOutOfBoundsException: 
  at java.util.ArrayList.add (ArrayList.java:468)
  at org.json.JSONArray.put (JSONArray.java:185)
  at com.applovin.impl.mediation.c.c.a (Unknown Source:102)
  at com.applovin.impl.sdk.e.i.b (Unknown Source:171)
  at com.applovin.impl.sdk.e.i.run (Unknown Source:46)
  at java.util.concurrent.ThreadPoolExecutor.processTask (ThreadPoolExecutor.java:1187)
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1152)
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:641)
  at java.lang.Thread.run (Thread.java:784)
```

And this one:
Error: java.lang.Error
```
at java.util.ArrayList.add (ArrayList.java:468)
  at org.json.JSONArray.put (JSONArray.java:185)
  at com.applovin.impl.mediation.c.c.a (Unknown Source:102)
  at com.applovin.impl.sdk.e.i.b (Unknown Source:171)
  at com.applovin.impl.sdk.e.i.run (Unknown Source:46)
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1162)
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:636)
  at java.lang.Thread.run (Thread.java:764)
```
I would like to have any more info, but this is all that Google Play / Firebase is showing us

**Comment by thomasmso** [comment]
Very helpful, appreciate it. You can expect a fixed SDK by Sunday.

**Comment by thomasmso** [comment]
We just released v11.2.0 which fixes these crashes.
